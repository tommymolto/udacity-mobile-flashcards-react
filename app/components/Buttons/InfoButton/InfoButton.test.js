/* global describe, it, expect */

import React from 'react';
import renderer from 'react-test-renderer';
import InfoButton from './InfoButton';

describe('InfoButton', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<InfoButton onPress={() => {}} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
