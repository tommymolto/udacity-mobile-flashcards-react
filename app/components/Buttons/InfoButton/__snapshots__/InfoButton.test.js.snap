// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InfoButton renders correctly 1`] = `
<View
  accessibilityComponentType={undefined}
  accessibilityLabel={undefined}
  accessibilityTraits={undefined}
  accessible={true}
  collapsable={undefined}
  hitSlop={undefined}
  isTVSelectable={true}
  nativeID={undefined}
  onLayout={undefined}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "opacity": 1,
    }
  }
  testID={undefined}
  tvParallaxProperties={undefined}
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Array [
        Object {
          "backgroundColor": "#BDC7CF",
          "color": "#000000",
          "fontSize": 22,
          "marginLeft": 5,
          "marginRight": 5,
          "marginVertical": 10,
          "paddingBottom": 10,
          "paddingTop": 10,
          "textAlign": "center",
        },
        undefined,
      ]
    }
  >
    Submit
  </Text>
</View>
`;
